src/conn/conn_api.c

```diff
@@ -1671,12 +1671,23 @@ __conn_statistics_config(WT_SESSION_IMPL *session, const char *cfg[])
 
 	if ((ret = __wt_config_subgets(
 	    session, &cval, "all", &sval)) == 0 && sval.val != 0) {
-		LF_SET(WT_CONN_STAT_ALL | WT_CONN_STAT_FAST);
+		LF_SET(WT_CONN_STAT_ALL |
+		    WT_CONN_STAT_CACHE_WALK | WT_CONN_STAT_FAST);
 		++set;
 	}
 	WT_RET_NOTFOUND_OK(ret);
 
 	if ((ret = __wt_config_subgets(
+	    session, &cval, "cache_walk", &sval)) == 0 && sval.val != 0) {
+		if (flags == 0)
+			WT_RET_MSG(session, EINVAL,
+			    "the value \"cache_walk\" is not compatible with "
```

Comment:
I was torn about this one. I agree that none is implicit, so the message isn't as useful as ideal to the user. I wanted to avoid having a growing list of options in the error message that match. I'll switch it to be consistent with the message for "clear"